commcare_connect/opportunity/forms.py

```diff
@@ -178,26 +194,39 @@ def __init__(self, *args, **kwargs):
 
         self.helper = FormHelper(self)
         self.helper.layout = Layout(
-            Row(Field("name")),
-            Row(Field("description")),
-            Row(Field("short_description")),
-            Fieldset(
-                "Learn App",
-                Row(Field("learn_app_domain")),
-                Row(Field("learn_app")),
-                Row(Field("learn_app_description")),
-                Row(Field("learn_app_passing_score")),
-                data_loading_states=True,
+            Row(
+                HTML("<div class='col-span-2'><h6 class='title-sm'>Opportunity Details</h6>  <span class='hint'>Add the details of the opportunity. All fields are mandatory.</span> </div>"),
+                Column(
+                    Field("name"),
+                    Field("short_description"),
+                    Field("description"),
```

Comment:
Can we make the description a full width on the form?

Reply:
These are all full width already.